### ISSUE TITLE
file:serialize json

### ISSUE THREAD

**Issue by ford-thomas-w** [eXist-db/exist#458]
I need to write out some json to the file system from an xml doc.  So, I tried the following:

```xquery
file:serialize(<root><item>thing</item></root>, "/path/to/data.json", ("method=json","media-type=application/json"))
```

In data.json the node is duplicated as such:
```javascript
{ "item" : "thing" } { "item" : "thing" }
```

My system info:

System Status
General
Uptime:	PT8H43M14.379S
eXist Version:	2.1.6-LTS
eXist Build:	20140616
eXist Home:	C:\eXist216
SVN Revision:	
Operating System:	Windows Server 2008 R2 6.1 amd64
File encoding:	UTF-8
Java
Vendor:	Oracle Corporation
Version:	1.7.0_51
Implementation:	Java HotSpot(TM) 64-Bit Server VM

**Comment by wolfgangmm** [comment]
The problem shows up on 2.2 as well, but it seems to be limited to file:serialize. I'll look into it.

**Comment by ford-thomas-w** [comment]
Any update on this? Was this fixed in 2.2?
